curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698440c54c8823cc60af8bc.md

```diff
@@ -29,7 +29,8 @@ Within the `ul` element nest three list items to display three things cats love:
 You should have three `li` elements. Each `li` element should have its own opening and closing tag.
 
 ```js
-assert($('li').length === 3 && code.match(/<\/li\>/g).length === 3);
+assert.lengthOf($('li').length, 3);
```

Comment:
can you also remove jQuery just you are here?
use `document.querySelectorAll` instead of `$`

Also, remove length if you use lengthOf